### DISCUSSION TITLE
How I can upload a pdf file into Chat window

### DISCUSSION THREAD

**Discussion by sonnt-dna** [langgenius/dify#4063] [General]
Hi everyone,
I want to upload a pdf file into chat window on mybot and apply RAG for any question with my pdf file. 
But I only upload images file.
Please to suggest me about can do it.
Thank you so much!

**Comment by xgmeng** [comment]
同求

**Comment by MyNameIsOka** [comment]
Would love to be able to upload different files than just images.
